### PR TITLE
feat(protocol-designer): labware name overlay shows only nickname or labware type

### PR DESCRIPTION
**Pull request by IanLondon**
# overview
addresses first part of #1281 (probably the smallest part)

# changelog
* labware name label shows only the nickname, or if no nickname, only the labware type
* "EDIT INGREDIENTS" -> "EDIT DETAILS" in labware overlay
* new `pen` icon

# review requests

https://s3-us-west-2.amazonaws.com/opentrons-protocol-designer/pd_labware-type-or-nickname-exclusive-1281-part-1/index.html

* Basic UI & code review
* Is Run App OK? @howisthisnamenottakenyet @pantslakz 

Run App deck now looks like:

image

PD deck looks like:

image

### CODE REVIEW

**Comment by IanLondon** [comment]
@howisthisnamenottakenyet added a link above

**Comment by pantslakz** [comment]
@IanLondon this will cause an issue for Run App - as its more important user knows what type of lab ware to place on deck for calibration.

File: app/src/components/DeckMap/LabwareItem.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ import {
   Icon,
   LabwareContainer,
   Plate,
-  type LabwareComponentProps
+  type LabwareComponentProps,
+  humanize
```
Comment (mcous) [inline app/src/components/DeckMap/LabwareItem.js]: A couple things with `humanize`:

• `ContainerNameOverlay` designs (seem to) dictate that `title` is always uppercase words
• Therefore, I don't think `humanize` should leak out of the components library here
    • ContainerNameOverlay should, for example, always call `humanize` on its title prop
• Also, `humanize` appears to be a duplicate of `lodash.startCase`
• Also also, seeing as it's always uppercase (with a CSS `text-transform`), perhaps `lodash.lowerCase` or `lodash.upperCase` would be more appropriate than `lodash.startCase`

Comment (IanLondon) [reply app/src/components/DeckMap/LabwareItem.js]: I didn't know that lodash has these methods - that's useful! I can do `humanize = startCase`

However, I think `title` needs to be treated differently depending on whether a labware is nicknamed or not -- eg I don't want to see `96-deep-well` I want `96 DEEP WELL`, but if the user nicknames a plate `Samples A-J` I don't want to eliminate the dashes and show `SAMPLES A J`, I just want to capitalize it: `SAMPLES A-J`.

Comment (IanLondon) [reply app/src/components/DeckMap/LabwareItem.js]: With that in mind, I just pushed cleanup for `humanize`:
    - rename to "humanizeLabwareType"
    - no funky import+export in protocol-designer/src/utils.js
    - use startCase as implementation
    - separately use startCase in formProcessing.js

**Pull request merged**